src/LiquidityPoolConverter.ts

```diff
@@ -204,12 +205,12 @@ class IConversionEvent {
 }
 
 function handleConversion(event: IConversionEvent): void {
-  const fromAmount = decimalize(event.fromAmount, event.fromToken)
-  const toAmount = decimalize(event.toAmount, event.toToken)
-  const conversionFee = decimalize(event.conversionFee, event.toToken)
-  const protocolFee = decimalize(event.protocolFee, event.toToken)
+  const toToken = Token.load(event.toToken.toHexString())
+  const fromAmount = decimal.fromBigInt(event.fromAmount, DEFAULT_DECIMALS)
+  const toAmount = decimal.fromBigInt(event.toAmount, DEFAULT_DECIMALS)
+  const conversionFee = decimal.fromBigInt(event.conversionFee, DEFAULT_DECIMALS)
+  const protocolFee = decimal.fromBigInt(event.protocolFee, DEFAULT_DECIMALS)
```

Comment:
protocolFee is in the toToken. I'm trying not to load entities that aren't strictly needed because it slows down syncing, especially with events like Conversion that are very frequent.

I want to refactor this so that toToken and fromToken are loaded only once (see above comment)

Reply:
I've changed this to use token decimals for everything, am marking as resolved